packages/replay-test/src/__tests__/replay-test.test.ts

```diff
@@ -360,6 +360,16 @@ test("can get global position and rotation of deeply nested textures", () => {
   expect(textures[0].props.rotation).toBe(0);
 });
 
+test("jumpToFrame throws last error", () => {
+  const { jumpToFrame, getTexture } = testSprite(Game(gameProps), gameProps, {
+    initInputs: {},
+  });
+
+  expect(() => jumpToFrame(() => getTexture("i-dont-exist"))).toThrowError(
+    `No textures found with test id "i-dont-exist"`
```

Comment:
It would be nice to still know there was a timeout. How about joining the error messages together? e.g:

```
Timeout of 1000 gameplay seconds reached on jumpToFrame with error:

No textures found with test id "i-dont-exist"
```

Reply:
Looks good to me! I'll make the change.